examples/01-basic/11-removing-block-option/App.tsx

```diff
@@ -0,0 +1,54 @@
+import "@blocknote/core/fonts/inter.css";
```

Comment:
I think these files still need to be removed and an `npm run build` is necessary

Reply:
So do you want me to remove the import of styles.css only or other files as well ?

Reply:
No I think you renamed the example, but kept all the old files?

Reply:
Oh sorry, my bad, that's a git issue, I forgot to stage some changes before commiting.